### PR TITLE
Changes to conform to new admin graphql endpoint

### PR DESCRIPTION
**Pull request by unclecheese**
Resolves: https://github.com/silverstripe/silverstripe-graphql/issues/158

### CODE REVIEW

File: _config/injector.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+---
+Name: admin-injector
```
Comment (maxime-rainville) [inline _config/injector.yml]: Can we had a bit of conditional logic here just to be on the safe side?
```yml
---
Name: admin-graphql
Only:
  moduleexists: silverstripe/graphql
---
```

Comment (maxime-rainville) [reply _config/injector.yml]: Same thing for the routes.

**Comment by maxime-rainville** [comment]
Travis is still testing against silverstripe/graphql 2.x-dev.

Wait for silverstripe/recipe-cms#18 and https://github.com/silverstripe/silverstripe-graphql/pull/173 to be merged and restart travis.

**Comment by robbieaverill** [comment]
Dependencies merged and builds restarted

**Pull request merged**